lib/config/themes.dart

```diff
@@ -263,83 +251,80 @@ abstract class FluffyThemes {
             iconSize: MaterialStateProperty.all(iconSize),
             iconColor: MaterialStateProperty.all(
               brightness == Brightness.light
-                ? LinagoraSysColors.material().onSurface
-                : LinagoraSysColors.material().onSurfaceDark,
+                  ? LinagoraSysColors.material().onSurface
+                  : LinagoraSysColors.material().onSurfaceDark,
             ),
-          )
+          ),
         ),
         iconTheme: IconThemeData(
           size: iconSize,
           color: brightness == Brightness.light
-            ? LinagoraSysColors.material().onBackground
-            : LinagoraSysColors.material().onBackgroundDark
+              ? LinagoraSysColors.material().onBackground
+              : LinagoraSysColors.material().onBackgroundDark,
         ),
         switchTheme: SwitchThemeData(
           materialTapTargetSize: MaterialTapTargetSize.shrinkWrap,
           overlayColor: MaterialStateProperty.resolveWith<Color?>(
             (Set<MaterialState> states) {
               if (states.contains(MaterialState.selected)) {
                 return brightness == Brightness.light
-                  ? LinagoraSysColors.material().primary
-                  : LinagoraSysColors.material().primaryDark;
+                    ? LinagoraSysColors.material().primary
+                    : LinagoraSysColors.material().primaryDark;
               } else {
                 return brightness == Brightness.light
-                  ? LinagoraSysColors.material().outline
-                  : LinagoraSysColors.material().outlineDark;
+                    ? LinagoraSysColors.material().outline
+                    : LinagoraSysColors.material().outlineDark;
               }
             },
           ),
           thumbColor: MaterialStateProperty.resolveWith<Color?>(
-            (Set<MaterialState> states) {
-              if (states.contains(MaterialState.selected)) {
-                return brightness == Brightness.light
+              (Set<MaterialState> states) {
+            if (states.contains(MaterialState.selected)) {
+              return brightness == Brightness.light
                   ? LinagoraSysColors.material().onPrimary
                   : LinagoraSysColors.material().onPrimaryDark;
-              } else {
-                return brightness == Brightness.light
+            } else {
+              return brightness == Brightness.light
                   ? LinagoraSysColors.material().outline
                   : LinagoraSysColors.material().outlineDark;
-              }
             }
-          ),
+          }),
```

Comment:
any wrong here?

Reply:
done, seem like dart analyzer do not work in this case